.github/workflows/release.yml

```diff
@@ -25,4 +25,4 @@ jobs:
         QUAY_TOKEN: ${{ secrets.QUAY_TOKEN }}
 
     - name: Build & push syncer image
-      run: make build-syncer-image push-syncer-image
+      run: make build-syncer-image v2 push-syncer-image
```

Comment:
Do we need v2 here as a make target?

Reply:
Ohh okay, in that case these needs to be interchanged. i.e `make v2 build-syncer-image push-syncer-image`

Reply:
sure! will update it.